src/Nri/Ui/BannerAlert/V4.elm

```diff
@@ -101,6 +101,7 @@ banner config alertMessage dismissMsg =
         [ css
             [ Css.displayFlex
             , Css.justifyContent Css.center
+            , Css.alignItems Css.center
```

Comment:
wohoo!  this fixed vertical alignment of the dismiss button, as the alert message gets longer.